File: ejercicio6tp1/ejercicio9.java
import java.util.Scanner;
import java.util.Random;

public class ejercicio9 {
    
    public static void main(String[] args){
    Scanner scanner = new Scanner(System.in);
    Random random = new Random();
    System.out.print("Ingrese un valor: ");
    int num = scanner.nextInt();
    int ran = random.nextInt();
    if (num == ran) System.out.println(num + " es igual a " + ran);
    else if (num > ran) System.out.println(num + " es mayor a " + ran);
        else System.out.println(num + " es menor a " + ran);
    scanner.close();
    }
}

File: TP1/TP1 python/python fede/ejercicio9.py
import random

num1 = int(input("Ingrese un número: "))
num2 = random.randint(1, 10)

if (num1==num2):
    print(num1, "es igual a", num2)
else:
    if (num1>num2):
        print(num1, "es mayor a", num2)
    else: print(num1, "es menor a", num2)